beamer/models/claim.py

```diff
@@ -29,6 +29,8 @@ def __init__(
     claimer_winning = State("ClaimerWinning", initial=True)
     # Challenger is winning
     challenger_winning = State("ChallengerWinning")
+    invalidated = State("Invalidated")
+    invalidated_l1_resolved = State("L1Invalidated")
```

Comment:
Nitpick: so far we had the state identifiers consistent with state names, but here we're diverging from that practice.
I'd suggest
```suggestion
    invalidated_l1_resolved = State("InvalidatedL1Resolved")
```